importanize/statements.py

```diff
@@ -135,60 +134,10 @@ def as_string(self):
                                         self.unique_leafs)))
             )
 
-    def formatted(self):
-        leafs = self.unique_leafs
-        string = self.as_string()
-        get_normalized = lambda i: map(
-            operator.attrgetter('normalized'), i
-        )
-
-        all_comments = (
-            self.comments + list(itertools.chain(
-                *list(map(operator.attrgetter('comments'), leafs))
-            ))
-        )
-
-        if len(all_comments) == 1:
-            string += '  # {}'.format(' '.join(get_normalized(all_comments)))
-
-        if any((len(string) > 80 and len(leafs) > 1,
-                len(all_comments) > 1)):
-            sep = '{}    '.format(self.file_artifacts.get('sep', '\n'))
-
-            string = 'from {} import ('.format(self.stem)
-
-            if self.comments:
-                string += '  # {}'.format(' '.join(
-                    get_normalized(self.comments)
-                ))
-
-            for leaf in leafs:
-                string += sep
-
-                first_comments = list(filter(
-                    lambda i: i.is_comment_first,
-                    leaf.comments
-                ))
-                if first_comments:
-                    string += sep.join(
-                        '# {}'.format(i)
-                        for i in get_normalized(first_comments)
-                    ) + sep
-
-                string += '{},'.format(leaf.as_string())
-
-                inline_comments = list(filter(
-                    lambda i: not i.is_comment_first,
-                    leaf.comments
-                ))
-                if inline_comments:
-                    string += '  # {}'.format(
-                        ' '.join(get_normalized(inline_comments))
-                    )
-
-            string += '{})'.format(self.file_artifacts.get('sep', '\n'))
-
-        return string
+    def formatted(self, formatter='IndentWithTabsFormatter'):
```

Comment:
I would rather pass here actual class so no import is necessary. Since formatters could be pretty generic, we might want to allow people to use custom formatters even if they are not in core importanize...